### PR TITLE
Port changes from releases/2021/4 to master

### PR DESCRIPTION
**Pull request by DariaMityagina**

### CODE REVIEW

**Comment by DariaMityagina** [comment]
@Maxim-Doronin could you please take a look?

**Comment by DariaMityagina** [comment]
@AntonDudchenko , @Maxim-Doronin could you please take a look?

**Comment by DariaMityagina** [comment]
@AntonDudchenko hi!
I ran another DLB job (# 803). It does not crash on the faster-vgg16 model due to timeouts, and it does not seem to experience any major performance degradation. 
FYI @Maxim-Doronin

**Review by AntonDudchenko — APPROVED** [review]

**Comment by AntonDudchenko** [comment]
@Maxim-Doronin could you take a look, please?

**Comment by DariaMityagina** [comment]
I also ran ie_tests_myriadx_ubuntu18 here.

File: inference-engine/cmake/vpu_dependencies.cmake [hunk 1]
```diff
@@ -6,19 +6,18 @@ include_guard(GLOBAL)
...
 # Default packages
 #
 
-set(FIRMWARE_PACKAGE_VERSION 1774)
+set(FIRMWARE_PACKAGE_VERSION 1875)
 set(VPU_CLC_MA2X8X_VERSION "movi-cltools-20.09.2")
 
 #
 # CMake variables to override default firmware files
-#
```
Comment (Maxim-Doronin) [inline inference-engine/cmake/vpu_dependencies.cmake:21]: Please, return it in next PRs

**Review by Maxim-Doronin — APPROVED** [review]

File: inference-engine/cmake/vpu_dependencies.cmake (hunk 1 above)
Comment (DariaMityagina) [reply inference-engine/cmake/vpu_dependencies.cmake:21]: Oops. Will do.

**Pull request merged**